pkg/stub/handler.go

```diff
@@ -207,10 +207,14 @@ func (h *Handler) VariableConfigChanged(srcfg *v1alpha1.SamplesResource, cm *cor
 	// Split will return an item of len 1 if skippedStreams is empty;
 	// verified via testing and godoc of method
 	if len(skippedStreams) > 0 {
+		// gotta trim any remaining space to get array size comparison correct
+		// a spling of say 'jenkins ' will create an array of size 2
+		skippedStreams = strings.TrimSuffix(skippedStreams, " ")
 		streams = strings.Split(skippedStreams, " ")
 	}
+
 	if len(streams) != len(srcfg.Spec.SkippedImagestreams) {
-		logrus.Printf("SkippedImageStreams changed from %s, processing %v", skippedStreams, srcfg)
+		logrus.Printf("SkippedImageStreams check 1 changed from %s, processing %v", skippedStreams, srcfg)
```

Comment:
leftover debug?

Reply:
(if you're intending to disambiguate the message from the other case, please make it something more human readable)